paasta_tools/long_running_service_tools.py

```diff
@@ -203,6 +205,12 @@ def get_replication_crit_percentage(self) -> int:
     def get_iam_role(self) -> str:
         return self.config_dict.get("iam_role", "")
 
+    def get_iam_role_provider(self) -> str:
+        return self.config_dict.get("iam_role_provider", "kiam")
+
+    def get_fs_group(self) -> Optional[int]:
```

Comment:
maybe lets move the default `33` and explanation comment here?

Reply:
`fs_group` might be useful besides the AWS pod identity-related workaround, but only for the workaround it need to be set to `33` by default.